client/src/pages/GalleryPage.js

```diff
@@ -1,54 +1,52 @@
 import React, { useState } from "react";
 import styled from "styled-components/macro";
+import PropTypes from "prop-types";
 import { useEffect } from "react";
```

Comment:
To make it a bit shorter:

```suggestion
import React, { useState, useEffect } from "react";
import styled from "styled-components/macro";
import PropTypes from "prop-types";
```